recipe.py

```diff
@@ -60,18 +63,25 @@ def remove_instruction(self, instruction):
 
 
 r1 = Recipe("Lasagne")
-# r1.set_description("Layers of pasta cheese and tomato sauce")
-# print(r1.get_description())
-#r1.set_no_servings(5)
+r1.set_description("Layers of pasta cheese and tomato sauce")
+print(r1.get_description())
+r1.set_no_servings(5)
 r1.set_ingredients("tomato")
 r1.set_ingredients("tomato")
 r1.set_ingredients("cheese")
 print(r1.get_ingredients())
 r1.remove_ingredient("cheese")
 print(r1.get_ingredients())
+r1.display_ingredients()
+
 
+window =tk.Tk()
```

Comment:
I think that all graphical formatting will be done in the recipe manager (or whatever calls the recipe manager).

Reply:
https://stackoverflow.com/questions/17466561/what-is-the-best-way-to-structure-a-tkinter-application gives a good indication of how to structure a project with tkinter.

Reply:
I will read the article and delete the graphical formatting